### PR TITLE
Search suggestions live view

### PR DESCRIPTION
**Pull request by alexbridge**

### CODE REVIEW

File: frontend/portals/SearchSuggestions/index.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import React, {
```
Comment (Carsten89) [inline frontend/portals/SearchSuggestions/index.jsx:1]: image

File: frontend/portals/SearchSuggestions/index.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  memo, useEffect, useMemo, useRef,
+} from 'react';
```
Comment (Carsten89) [inline frontend/portals/SearchSuggestions/index.jsx:3]: shouldn't the search suggestions be scrollable?

File: frontend/portals/SearchSuggestions/index.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
```
Comment (Carsten89) [inline frontend/portals/SearchSuggestions/index.jsx:5]: loading spinner is hidden under TabBar
image

File: frontend/portals/SearchSuggestions/index.jsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import { useRoute } from '@shopgate/engage/core';
```
Comment (Carsten89) [inline frontend/portals/SearchSuggestions/index.jsx:6]: there is a small gap where the underlying content shows through
image

File: frontend/portals/SearchSuggestions/index.jsx (hunk 1 above)
Comment (alexbridge) [reply frontend/portals/SearchSuggestions/index.jsx:1]: It is a backend lie . I will put a check for actual product count

File: frontend/portals/SearchSuggestions/index.jsx (hunk 2 above)
Comment (alexbridge) [reply frontend/portals/SearchSuggestions/index.jsx:3]: Hm. Sure. They are scrollable on Android. Isnt it on iOs?

File: frontend/portals/SearchSuggestions/index.jsx (hunk 1 above)
Comment (Carsten89) [reply frontend/portals/SearchSuggestions/index.jsx]: did you add it?
image

File: frontend/portals/SearchSuggestions/index.jsx (hunk 2 above)
Comment (Carsten89) [reply frontend/portals/SearchSuggestions/index.jsx]: tested on desktop chrome. couldn't also find any css that should do the horizontal scrolling of the word suggestions

File: frontend/portals/SearchSuggestions/index.jsx (hunk 1 above)
Comment (Carsten89) [reply frontend/portals/SearchSuggestions/index.jsx]: loading spinners still spins forever

Comment (alexbridge) [reply frontend/portals/SearchSuggestions/index.jsx]: I did: https://github.com/shopgate-professional-services/ext-search-suggestions-extended/pull/1/files#diff-655a2290f3c38806b891411d4e6fa0cdR28

What is response? I guess count is 566 but actual 1 item. I will add 1 more

**Pull request merged**